### ISSUE TITLE
Request CSS animation-fill-mode

### ISSUE THREAD

**Issue by rafadsm** [mikke89/RmlUi#506]
Could we see the animation-fill-mode css property in the future?
I believe it shouldn't be something so complex to implement
For details: https://developer.mozilla.org/en-US/docs/Web/CSS/animation-fill-mode

**Comment by mikke89** [comment]
Yeah, this has pretty much been on my todo-list since animation support was added. I haven't heard anyone request it until now, so that's why it's been left on the table. I'll try to get around to it at some point, otherwise a pull request would be appreciated.
